projects2.0/FAO/Fish/Fish_Reftbl_RefDatasets/parser.js

```diff
@@ -1,12 +1,9 @@
 const path = require('path');
 // @TODO update require. when we export this method - we can connect it from index.js
-const {
-  parseDirectoryFiles,
-  getHeaders
-} = require('../../../../dist/index.cjs');
+const { parseDirectoryFiles, parseCsv } = require('../../../../dist/index.cjs');
```

Comment:
maybe we can also fix problem with this path?
it drives me crazy

Reply:
we can solve this with `app-root-path` package but the code in parser will be like this

`const appRoot = require('app-root-path');

const { parseDirectoryFiles, parseCsv } = require(appRoot + '/dist/index.cjs');`

Reply:
hmm. you make a decision

Reply:
do you think it will confuse other interns or no?